core/contracts/oracle/implementation/Voting.sol

```diff
@@ -374,7 +389,8 @@ contract Voting is Testable, Ownable, OracleInterface, VotingInterface, Encrypte
         )) == voteSubmission.commit, "Revealed data != commit hash");
         delete voteSubmission.commit;
 
-        // Lock in round variables including snapshotId and inflation rate
+        // Lock in round variables including snapshotId and inflation rate. Not that this will only execute a snapshot
+        // if the `snapshotCurrentRound` function was not already called for this round.
         _freezeRoundVariables(roundId);
```

Comment:
This is now exactly as it was before. You can now snapshot as the first revealer _or_ by calling `snapshotCurrentRound`.

Reply:
So do you anticipate in production that UMA calls `snapshotCurrentRound` before each reveal round, but we don't technically enforce this via code?